rdr_service/genomic/genomic_state_handler.py

```diff
@@ -247,6 +247,12 @@ def transition_function(self, signal):
             return ResultsWorkflowState.CVL_W3SR
 
 
+class W3NSState(GenomicStateBase):
```

Comment:
I'm good with this for now, just noting that we'll probably want to revisit the Secondary Confirmation loop when we're designing the reconciliation process. Looking at the workflow diagram, the state transitions could are a little messy.